packages/dropdowns.next/src/views/combobox/StyledValue.ts

```diff
@@ -39,7 +40,7 @@ export const StyledValue = styled.div.attrs({
       return 'default';
     }
 
-    return props.isEditable ? 'text' : 'pointer';
+    return props.isEditable && !props.isAutocomplete ? 'text' : 'pointer';
```

Comment:
Since `isAutocomplete` now triggers expansion, we want the cursor to be a pointer